### ISSUE TITLE
Publish as android library

### ISSUE THREAD

**Issue by yschimke** [google/mobly-bundled-snippets#159]
Suggestion to publish as an android library, instead of building an application.

Apps could use this and mobly-snippet-lib in their own codebase in an artifact published to maven central.

**Comment by xpconanfan** [comment]
I think we already publish to Android maven, no?

**Comment by xianyuanjia** [comment]
mobly-snippet-lib is published to Maven, but mobly-bundled-snippets is not.

**Comment by yschimke** [comment]
Yeah, my issue is that I'd like to use these for different apps, but with this project being targetted to a specific package, it isn't easy to reuse.

I'd like a clean dependency I can pull in, and only have additional snippets in my app specific mobly module.

**Comment by xpconanfan** [comment]
Yeah, it would be ideal if we can pull the individual snippet classes as needed. But is that feasible in maven?

**Comment by yschimke** [comment]
This might also remove the concern around targetSdk in https://github.com/google/mobly-bundled-snippets/pull/158, since that shouldn't apply to libraries.
